tests/integration/targets/pipx/tasks/main.yml

```diff
@@ -3,10 +3,28 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-- name: install pipx
-  pip:
-    name: pipx
-    extra_args: --user
+- name: Determine pipx level
+  block:
+    - name: Install pipx>=1.7.0
+      pip:
+        name: pipx>=1.7.0
+        extra_args: --user
+    - name: Ensure path
+      ansible.builtin.command: "{{ ansible_python_interpreter }} -m pipx ensurepath"
```

Comment:
This seems to cause problems on FreeBSD; it seems to hang there.

Reply:
Yup, I noticed. I am trying to get my FreeBSD VMs running in vagrant again to test this locally, bumping into C compiler issues.

Reply:
Good luck :)